### PR TITLE
feat: add docstring to get_pollution_history function

### LINKED ISSUE #1: Docstring for get_pollution_history()

### PR DESCRIPTION
**Pull request by mel-liow**
- [X] Add docstring for get pollution history

### CODE REVIEW

File: src/rpollution.R [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#' 
+#' Given a specified date range, the function makes an API request 
+#' to the OpenWeather Air Pollution API and fetches historic 
+#' pollution data for a given location.
+#' 
+#' Note: Historical data is accessible from 27th November 2020
+#'
+#' @param start_date A UNIX timestamp, e.g. 1606488670
+#' @param end_date A UNIX timestamp, e.g. 1606747870
+#' @param lat Latitude coordinate for a given location as a float
```
Comment (danfke) [inline src/rpollution.R]: Should be float or integer

**Review by danfke — CHANGES_REQUESTED** [review]
I'm writing the docstring for my function using the workflow in https://r-pkgs.org/whole-game.html#write-the-first-function. Using this workflow will automatically update other files and also creates a file for each function because of this each function should get its own file. See my pull request for an example

**Review by christopheralex — CHANGES_REQUESTED** [review]
I think there has to be changes in the man file for the docstring as well . So as dan said , would be better if you follow the IAT2 guide.

**Review by danfke — APPROVED** [review]
Looks good thanks Mel!

**Review by christopheralex — APPROVED** [review]

**Pull request merged**